### DISCUSSION TITLE
validate_bban flag

### DISCUSSION THREAD

**Discussion by michaelgfeldman** [mdomke/schwifty#222] [General]
Hi! 
Am i understanding this right that validate_bban (the country specific checksums) option only works for Belgium, Germany and Italy? If no, where can I get the full list of supported countries?
Thanks for your work!

**Comment by mdomke** [comment]
The BBAN validation should work for
* Albania
* Belgium
* Bosnia and Herzegovina
* Montenegro
* North Macedonia
* Portugal
* Serbia
* Slovenia
* Slovakia
* East Timor
* Mauretania
* Tunesia
* Czech Republic
* Estonia
* Finland
* France
* Monaco
* Germany
* Iceland
* Italy
* San Marino
* Norway
* Poland
* Spain

You can have a look [here](https://github.com/mdomke/schwifty/blob/main/schwifty/checksum), but I guess I could improve the documentation in that regard.
